<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COM_port</class>
 <widget class="QWidget" name="COM_port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>COM_pot_text.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="COMportlineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM 3</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:flat {   border: none; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>connect_button.png</normaloff>connect_button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1000</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
